coins/zcash_testnet.json

```diff
@@ -5,8 +5,8 @@
     "requireShielding": true,
     "payFoundersReward": true,
     "percentFoundersReward": 20,
-    "maxFoundersRewardBlockHeight": 849999,
-    "foundersRewardAddressChangeInterval": 17709.3125,
+    "maxFoundersRewardBlockHeight": 839999,
+    "foundersRewardAddressChangeInterval": 17500,
```

Comment:
Confirming mining zcash_testnet blocks is still possible with this change, as a mined a block was mined in my testnet pool.